### PR TITLE
rework index page and overview

### PR DESCRIPTION
**Pull request by ru-fu**
- Moving some content from the index page to the overview
- Adding links to important information to the start page
- Expanding the information about Anbox Cloud vs. Anbox Cloud Appliance

Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

File: explanation/anbox-cloud.md [hunk 1]
```diff
@@ -13,47 +17,61 @@ The Anbox Cloud Appliance is a self-contained, single-machine deployment variant
...
 The regular Anbox Cloud uses [Juju](https://juju.is/) for deployment and operations. It provides rich features and is ready made for a large scale deployment.
 
+See the following table for a comparison of features for the different variants:
+
+| Feature | Anbox Cloud Appliance | Anbox Cloud |
+|---------|-----------------------|-------------|
+| [Streaming capabilities](https://discourse.ubuntu.com/t/streaming-android-applications/17769) | ✓ | ✓ |
+| [Web dashboard](https://discourse.ubuntu.com/t/web-dashboard/20871) | ✓ | ✓ |
+| [Android API version](https://discourse.ubuntu.com/t/provided-images/24185) | 10, 11 | 10, 11 |
+| [Long term security updates](https://discourse.ubuntu.com/t/supported-versions/21046) | ✓ | ✓ |
```
Comment (ru-fu) [inline explanation/anbox-cloud.md:27]: Do we provide that? The link says we don't ...

File: explanation/anbox-cloud.md [hunk 2]
```diff
@@ -13,47 +17,61 @@ The Anbox Cloud Appliance is a self-contained, single-machine deployment variant
...
+| [Community support](https://discourse.ubuntu.com/c/anbox-cloud/) | ✓ | ✓ |
+| [Vendor support](https://anbox-cloud.io/contact-us) | - | ✓ |
+| [Monitoring](https://discourse.ubuntu.com/t/monitor-anbox-cloud/24338) | - | ✓ |
```
Comment (ru-fu) [inline explanation/anbox-cloud.md:30]: Is the link correct for what we mean with monitoring here?

File: explanation/anbox-cloud.md [hunk 3]
```diff
@@ -13,47 +17,61 @@ The Anbox Cloud Appliance is a self-contained, single-machine deployment variant
...
+
 Which of both variants you choose depends on your needs. The appliance is well suited for quick prototyping and development or small scale deployments, whereas the regular Anbox Cloud is best to scale big.
 
+[note type="information" status="Tip"]
+We recommend to always start with the Anbox Cloud Appliance for first tests. You can then expand to a full Anbox Cloud installation later.
```
Comment (ru-fu) [inline explanation/anbox-cloud.md:35]: Do we actually? :wink:

File: index.md [hunk 4]
```diff
@@ -1,30 +1,39 @@
...
 
-For containerization of Android, Anbox Cloud uses the well established and secure container hypervisor [LXD](https://linuxcontainers.org/). LXD is secure by design, scales to a large number of containers and provides advanced resource management for hosted containers.
+## Core features
 
-Also have a look at the [official Anbox Cloud website](https://anbox-cloud.io/) for more information.
+* Simple and straightforward deployment
+* Management of the container and application life cycle while optimising for high density, performance and fast container boot times
+* Platform integration tools to allow, for example, integration of existing streaming solutions
+* Support for both x86 and Arm64 hardware
+* Integration with 3rd party solutions for binary translation solutions on Arm64-only hardware
```
Comment (ru-fu) [inline index.md:11]: I don't understand what we're saying here ... any way to make it clearer?

File: explanation/anbox-cloud.md (hunk 1 above)
Comment (morphis) [reply explanation/anbox-cloud.md:27]: For Anbox Cloud we don't but for Ubuntu we do. I think lets rephrase this as just "Security updates"

File: explanation/anbox-cloud.md (hunk 2 above)
Comment (morphis) [reply explanation/anbox-cloud.md:30]: Yes, we will need to adjust the content behind this but that looks good.

File: explanation/anbox-cloud.md (hunk 3 above)
Comment (morphis) [reply explanation/anbox-cloud.md:35]: Yes! :-)

File: index.md (hunk 4 above)
Comment (morphis) [reply index.md:11]: Lets drop this item. It's not true anymore.

**Review by morphis — CHANGES_REQUESTED** [review]

File: explanation/anbox-cloud.md (hunk 1 above)
Comment (ru-fu) [reply explanation/anbox-cloud.md:27]: I changed the link to https://ubuntu.com/support now.

**Review by morphis — APPROVED** [review]
LGTM
